build.gradle

```diff
@@ -232,6 +239,12 @@ configurations {
 }
 
 dependencies {
+    implementation('net.minidev:json-smart:2.4.9') {
+        exclude group: 'net.minidev', module: 'json-smart'
+    }
+    // Example dependencies
+    implementation 'biz.aQute.bnd:biz.aQute.bndlib:5.1.2'
```

Comment:
Don't add new GATK dependencies here unless absolutely necessary

Reply:
Again dnsjava is used from hadoop client.

Reply:
@droazen . Let us know if any more modifications required